Panic if no permissions in log directory

``` ./vuls report --debug
panic: open /var/log/vuls/cve-dictionary.log: permission denied

goroutine 1 [running]:
github.com/inconshreveable/log15.must(...)
        /home/ubuntu/go/pkg/mod/github.com/inconshreveable/log15@v0.0.0-20180818164646-67afb5ed74ec/handler.go:340
github.com/inconshreveable/log15.muster.FileHandler(0xc000240b00, 0x20, 0x19d76c0, 0x169aef0, 0x20, 0x0)
        /home/ubuntu/go/pkg/mod/github.com/inconshreveable/log15@v0.0.0-20180818164646-67afb5ed74ec/handler.go:348 +0x8a
github.com/kotakanbe/go-cve-dictionary/log.SetLogger(0x15fa9c6, 0xd, 0x100)
        /home/ubuntu/go/pkg/mod/github.com/kotakanbe/go-cve-dictionary@v0.4.1/log/log.go:40 +0x227
github.com/future-architect/vuls/commands.(*ReportCmd).Execute(0xc0000ee2a0, 0x19fd5a0, 0xc0000c2010, 0xc0000c8840, 0x0, 0x0, 0x0, 0x0)
        /home/ubuntu/go/src/github.com/future-architect/vuls/commands/report.go:195 +0xc3
github.com/google/subcommands.(*Commander).Execute(0xc0000ca000, 0x19fd5a0, 0xc0000c2010, 0x0, 0x0, 0x0, 0xc000428f38)
        /home/ubuntu/go/pkg/mod/github.com/google/subcommands@v1.0.1/subcommands.go:142 +0x2f9
github.com/google/subcommands.Execute(...)
        /home/ubuntu/go/pkg/mod/github.com/google/subcommands@v1.0.1/subcommands.go:420
main.main()
        /home/ubuntu/go/src/github.com/future-architect/vuls/main.go:37 +0x387```

--- a/go.mod
+++ b/go.mod
@@ -32,7 +32,7 @@ require (
 	github.com/knqyf263/go-rpm-version v0.0.0-20170716094938-74609b86c936
 	github.com/knqyf263/go-version v1.1.1
 	github.com/knqyf263/gost v0.1.3
-	github.com/kotakanbe/go-cve-dictionary v0.4.1
+	github.com/kotakanbe/go-cve-dictionary v0.4.2
 	github.com/kotakanbe/go-pingscanner v0.1.0
 	github.com/kotakanbe/goval-dictionary v0.2.5
 	github.com/kotakanbe/logrus-prefixed-formatter v0.0.0-20180123152602-928f7356cb96

